plan.lua

```diff
@@ -104,22 +129,9 @@ end
 --------------------------------------
 function plan_class:adjust_building_info(plan_pos, node)
 	-- adjust min/max position information
-	if not self.data.max_pos.y or plan_pos.y > self.data.max_pos.y then
-		self.data.max_pos.y = plan_pos.y
-	end
-	if not self.data.min_pos.y or plan_pos.y < self.data.min_pos.y then
-		self.data.min_pos.y = plan_pos.y
-	end
-	if not self.data.max_pos.x or plan_pos.x > self.data.max_pos.x then
-		self.data.max_pos.x = plan_pos.x
-	end
-	if not self.data.min_pos.x or plan_pos.x < self.data.min_pos.x then
-		self.data.min_pos.x = plan_pos.x
-	end
-	if not self.data.max_pos.z or plan_pos.z > self.data.max_pos.z then
-		self.data.max_pos.z = plan_pos.z
-	end
-	if not self.data.min_pos.z or plan_pos.z < self.data.min_pos.z then
+  self:adjust_bound_pos(plan_pos)
+
+  if not self.data.min_pos.z or plan_pos.z < self.data.min_pos.z then
```

Comment:
This lines are moved to adjust_bound_pos. Is there a reason to keep the lines redundant in adjust_building_info?

Reply:
Sorry, my mistake.